tests/unit/flow/test_flow_with_sse.py

```diff
@@ -0,0 +1,9 @@
+from jina.flow import Flow
+
+
+def test_flow_with_sse():
+    f = Flow(logserver=True).\
+        add(uses='BaseExecutor', parallel=1).add(uses='BaseExecutor', parallel=2).build()
+    with f:
```

Comment:
this test is blocked forever without the changes

Reply:
Still does not solve, when using `pokedex-with-bit` with logserver there is still a deadlock